docs/SRS-Volere/SRS.tex

```diff
@@ -77,22 +79,44 @@ \subsection{Maintenance Users and Service Technicians}
 
 \section{Mandated Constraints}
 \subsection{Solution Constraints}
-\lips
+\textbf{Mobile Device}\\
+\underline{Description}: User will interact with the system through an app on their mobile device (smartphone).\\
+\underline{Rational}: Mobile devices are widly used and ubiquitous. While at a grocery store, users may need access to the system.\\
```

Comment:
- Mobile devices are _widely_ used
- Also, is it supposed  to be "rationale" instead of rational? If yes, then all the points need to change to rationale